rust-toolchain

```diff
@@ -1 +1 @@
-stable-2020-10-08
+stable-2021-03-25
```

Comment:
1.51.0

Reply:
Alternatively, I think we might consider removing this file altogether. For stable, it's not super valuable, and it might be beneficial to use not exactly the specified toolchain. For example, I use `rustup default beta`, so that, when hacking of the projects, I can notice minor breakages.

Reply:
Basically, remove specified toolchain from the `near-sdk-rs`, but still use rust version for the `contract-builder`?

Reply:
Yup